commandlineparser/CMakeLists.txt

```diff
@@ -28,7 +27,12 @@ PUBLIC_HEADER DESTINATION ${_include}
 install(FILES ${header_files} DESTINATION ${_include})
 
 #install the export group
-install(EXPORT cmd_lib DESTINATION ${_lib})
+install(EXPORT cmd_lib
+  FILE
+    cmd_lib.cmake
+  DESTINATION
+  "lib/cmake/cmd_lib"
+)
+
+install(FILES cmd_lib-config.cmake DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/cmake/cmd_lib)
```

Comment:
{PROJECT_NAME}

Reply:
This is not a solution. as This is config file and should not have and Project related variable